src/nvim/message.c

```diff
@@ -3217,14 +3209,12 @@ int vim_vsnprintf(char *str, size_t str_m, char *fmt, va_list ap, typval_T *tvs)
         min_field_width = uj;
       }
 
-      /* parse precision */
+      // parse precision
       if (*p == '.') {
         p++;
         precision_specified = 1;
         if (*p == '*') {
-          int j;
-
-          j = tvs != NULL ? tv_nr(tvs, &arg_idx) : va_arg(ap, int);
+          int j = tvs != NULL ? tv_nr(tvs, &arg_idx) : va_arg(ap, int);
```

Comment:
Should be consistent with line 3195 where `int j = tvs ? ...` instead of `int j = tvs != NULL ? ...`